packages/ingest-mongodb-public/src/sources/mongodbDotCom/webSources.ts

```diff
@@ -0,0 +1,233 @@
+      "https://www.mongodb.com/resources/basics/artificial-intelligence/generative-ai-use-cases",
+      "https://www.mongodb.com/resources/basics/artificial-intelligence/ai-agents",
+      "https://www.mongodb.com/resources/basics/vector-index",
+      "https://www.mongodb.com/resources/basics/prompt-engineering",
+      "https://www.mongodb.com/resources/basics/predictive-ai",
+      "https://www.mongodb.com/resources/basics/reinforcement-learning",
+      "https://www.mongodb.com/resources/basics/artificial-intelligence/machine-learning-healthcare",
+      "https://www.mongodb.com/resources/basics/named-entity-recognition",
+      "https://www.mongodb.com/resources/basics/langchain",
+      "https://www.mongodb.com/resources/basics/quantum-machine-learning",
+      "https://www.mongodb.com/resources/basics/ann-search",
+      "https://www.mongodb.com/resources/basics/vector-embeddings",
+      "https://www.mongodb.com/resources/basics/self-supervised-learning",
+      "https://www.mongodb.com/resources/basics/vector-search",
+      "https://www.mongodb.com/resources/basics/unified-modeling-language",
+      "https://www.mongodb.com/resources/basics/ai-hallucinations",
+      "https://www.mongodb.com/resources/basics/ai-in-finance",
+      "https://www.mongodb.com/resources/basics/what-is-stream-processing",
+      "https://www.mongodb.com/resources/basics/cognitive-search",
+      "https://www.mongodb.com/resources/basics/foundation-models",
+      "https://www.mongodb.com/resources/basics/ai-stack",
+      "https://www.mongodb.com/resources/products/compatibilities/kubernetes",
+      "https://www.mongodb.com/resources/products/capabilities/stored-procedures",
+      "https://www.mongodb.com/resources/basics/cloud-explained/iaas-infrastructure-as-a-service",
+      "https://www.mongodb.com/resources/basics/databases/cloud-databases/cloud-migration",
+      "https://www.mongodb.com/resources/basics/unstructured-data/tools",
+      "https://www.mongodb.com/resources/basics/system-of-insight",
+      "https://www.mongodb.com/resources/basics/databases/middleware",
+      "https://www.mongodb.com/resources/basics/disaster-recovery",
+      "https://www.mongodb.com/resources/basics/cloud-explained/business-intelligence-bi-tools",
+      "https://www.mongodb.com/resources/basics/observability",
+      "https://www.mongodb.com/resources/basics/role-based-access-control",
+      "https://www.mongodb.com/resources/basics/real-time-payments",
+      "https://www.mongodb.com/resources/basics/databases/cloud-databases/free-cloud-database",
+    ],
+    staticMetadata: {
+      type: "Resources",
+    },
+  },
+  {
+    name: "web-misc",
+    urls: [
+      "https://www.mongodb.com",
+      "https://www.mongodb.com/atlas",
+      "https://www.mongodb.com/try/download/community",
+      "https://www.mongodb.com/pricing",
+    ],
+    staticMetadata: {
+      type: "Miscellaneous Web",
+    },
```

Comment:
```suggestion
```

can remove since this doesn't provide real semantic value to the vector search where it's consumed